src/staff/Staff.stories.tsx

```diff
@@ -1,34 +1,39 @@
 import React from "react";
 import { Story } from '@storybook/react/types-6-0';
 
-import Staff, {StaffProps} from "./Staff";
+import Staff, { StaffProps } from "./Staff";
 
 export default {
   component: Staff,
   title: 'Staff',
 };
 
-const StaffTemplate: Story<StaffProps> = (args) => <Staff {...args} />;
+const StaffTemplate: Story<StaffProps> = ( args ) => <Staff {...args} />;
```

Comment:
I use prettier to format styling and is resetting most of the files. It would be good to add a prettier config so we avoid code style diffs.

Defaults in JazzPianoAcademy should be good enough for this repo as well.

Would you mind adding a prettier config and run it on all the files?

Reply:
Sure, Amy! Will update on that! I've used AirBnB ESLint format!

Reply:
I've updated to prettier configuration to the project in e204311972269c7afcc78c494796eaae78fc659c commit, Amy!